lib/browser/guest-view-manager.ts

```diff
@@ -245,23 +260,25 @@ const attachGuest = function (event, embedderFrameId, elementInstanceId, guestIn
   guestInstance.elementInstanceId = elementInstanceId;
 
   watchEmbedder(embedder);
-
-  webViewManager.addGuest(guestInstanceId, elementInstanceId, embedder, guest, webPreferences);
+  if (webViewManager) {
```

Comment:
We should assert that `webViewManager` exists here instead of truthy checking, we want to do `webViewManager!` and let it error if it doesn't exist, it's a theoretical impossible case.

Reply:
Thanks for the guidance on how to treat this!